### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by denysdovhan**
Here are few upgrades for **how-to-markdown**. @workshopper/how-to-markdown-leads pleases check.

### CODE REVIEW

File: test/blockquotes/valid.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Blockquotes
+
+> To be, or not to be, that is the question.
+> - William Shakespeare
```
Comment (wooorm) [inline test/blockquotes/valid.md]: The citation get’s turned into a list-item in this case!

# Blockquotes

> To be, or not to be, that is the question.
> - William Shakespeare

Comment (denysdovhan) [reply test/blockquotes/valid.md]: 😱 needs to be fixed

Comment (itzsaga) [reply test/blockquotes/valid.md]: 👍 e705647.

**Comment by denysdovhan** [comment]
What do you think? Could be merged?

**Comment by wangding** [comment]
chinese menu is ok!


chinese menu

**Comment by itzsaga** [comment]
I think it's good to go.

**Comment by denysdovhan** [comment]
Published as `1.4.0`!

**Pull request merged**